sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue

```diff
@@ -12,12 +12,14 @@
       ></formio>
     </template>
     <template v-slot:footer>
-      <v-btn variant="outlined" :color="COLOR_BLUE" @click="dialogClosed">
-        Cancel
-      </v-btn>
-      <v-btn class="primary-btn-background" @click="approveProgram">
-        Approve now
-      </v-btn>
+      <v-row class="m-0 p-0">
+        <v-btn variant="outlined" :color="COLOR_BLUE" @click="dialogClosed">
+          Cancel
+        </v-btn>
+        <v-btn class="primary-btn-background" @click="approveProgram">
```

Comment:
The style primary-btn-background was created while using alpha  version. We can get rid of primary-btn-background with color="primary"

Reply:
okay, Thanks for that. since there are a lot of files using `primary-btn-background`. I will fix this in the next PR